pkg/controller/pod/pod_controller.go

```diff
@@ -93,7 +93,7 @@ func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result
 	err = r.Get(context.TODO(), request.NamespacedName, instance)
 	if err != nil {
 		if kuberneteserror.IsNotFound(err) {
-			err := r.poolManager.ReleasePodMac(fmt.Sprintf("%s/%s", request.Namespace, request.Name))
+			err := r.poolManager.ReleaseAllMacsOnPodDelete(fmt.Sprintf("pod/%s/%s", request.Namespace, request.Name))
```

Comment:
The fact that the pod was deleted is stored in the `if` above this call. The function can be called any time I want. The name of the called function reflects the context it is called in. Isn't the suffix redundant? Or worse, misleading?